### PR TITLE
Support SCRAM-SHA-256 password hashing on 10+

### PR DESCRIPTION
**Pull request by kravietz**
Attempt to fix https://github.com/dev-sec/postgres-baseline/issues/24

Please review this for syntax correctness as I don't have plenty of experience with this.

### CODE REVIEW

File: controls/postgres_spec.rb [hunk 1]
```diff
@@ -157,11 +157,24 @@
   impact 1.0
   title 'Use salted MD5 to store postgresql passwords'
   desc 'Store postgresql passwords in salted hash format (e.g. salted MD5).'
-  describe postgres_session(USER, PASSWORD).query('SELECT passwd FROM pg_shadow;') do
-    its('output') { should match(/^md5\S*$/) }
-  end
-  describe postgres_conf(POSTGRES_CONF_PATH) do
-    its('password_encryption') { should eq 'on' }
+  describe command('psql -V') do
```
Comment (chris-rock) [inline controls/postgres_spec.rb]: Can we make this a ruby function? e.g. `def pg_version()` that returns 9 or 10 as the version? Then the test could look much simpler:
```
expected_passwd = /^md5\S*$/
expected_password_encryption = 'on'

if pg_version() == 10
  expected_passwd = /^scram-sha-256\S*$/
  expected_password_encryption = 'scram-sha-256'
end

describe postgres_session(USER, PASSWORD).query('SELECT passwd FROM pg_shadow;') do
    its('output') { should match(expected_passwd) }
end
describe postgres_conf(POSTGRES_CONF_PATH) do
    its('password_encryption') { should eq expected_password_encryption }
end
```

Comment (kravietz) [reply controls/postgres_spec.rb]: @chris-rock I'm afraid my knowledge of Ruby is asymptotic to zero :)

**Comment by chris-rock** [comment]
@kravietz Can you fix the DCO, I'll work on top of your PR to add the changes then

**Comment by kravietz** [comment]
Thanks @chris-rock sign-off done!

**Comment by chris-rock** [comment]
@kravietz Thank you. My PR is up now  #31

**Pull request merged**